features/utils/CommonConstants.py

```diff
@@ -5,4 +5,5 @@
 FIELD_PASS='#loginpassword'
```

Comment:
these elements should not be mapped here, in common constants you can have texts you enter in the fields.
but the fields should be mapped to the pages they belong to.

Reply:
all the elements you have mapped here should be moved to the pages. and all the texts you wrote on the contact page and on login page, should be here.